guava/pom.xml

```diff
@@ -117,11 +122,33 @@
             <compilerArgs>
               <arg>-Xbootclasspath/p:${annotatedJdk}</arg>
               <arg>${addtionalargs}</arg>
+              <arg>-AsuppressWarnings=${checkerframework.suppress}</arg>
               <arg>-AuseDefaultsForUncheckedCode=source,bytecode</arg>
               <arg>-Awarns</arg>
               <arg>-AprintErrorStack</arg>
             </compilerArgs>
         </configuration>
+        <executions>
+          <!-- Runs the Index Checker on selected packages. -->
+          <!-- Because Index Checker warnings are suppressed during compilation, this additional phase
+               runs the Index Checker on packages that are already annotated. -->
+          <execution>
+          	<id>check-index</id>
```

Comment:
Minor: please don't use tabs.  These are the only two in the file (on this line and the next).

There are a few other scattered tabs in the diffs too.